.github/workflows/linux-libc++-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Linux (libc++) Build CI
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - README.md
+    tags:
+      - "v*.*.*"
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Build play-cpp-sdk library
+        run: CXX=clang++-10 CXXFLAGS=-stdlib=libc++ cargo build --package play-cpp-sdk --release
+
+      - name: Build demo project
+        working-directory: demo
+        run: make CXX=clang++-10
+
+      - name: Pack binaries and bindings
+        run: |
+          PLATFORM="$(uname -s)_x86_64"
+          BUILD_TYPE="tarball"
+          mkdir -p install
+          cp -r demo/sdk install
+          cp ./LICENSE install/sdk
+          cp ./CHANGELOG.md install/sdk
+          cd install
+          tar zcvf ../play_cpp_sdk_libc++_${PLATFORM}.tar.gz *
+          cd ..
+          sha256sum *.tar.gz > "checksums-$PLATFORM.txt"
+          echo "release_file=play_cpp_sdk_libc++_$PLATFORM.tar.gz" >> $GITHUB_ENV
+          echo "checksum_file=checksums-$PLATFORM.txt" >> $GITHUB_ENV
+          cat $GITHUB_ENV
+
+      - name: Upload binaries and bindings to Release
+        uses: softprops/action-gh-release@v1
```

Comment:
not sure if this one needs extra (write) permissions on the token

Reply:
It seems it is fine. Same as before, after tagging, file will be uploaded as a draft release.